spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/endpoint/web/CorsEndpointProperties.java

```diff
@@ -46,7 +46,7 @@
 
 	/**
 	 * Comma-separated list of origin patterns to allow. Unlike allowed origins which only
-	 * supports '*', origin patterns are more flexible (for example
+	 * support '*', origin patterns are more flexible (for example
```

Comment:
I think "supports" is correct here.

Reply:
@wilkinsona "origins" is plural, so I thought it needs to be "support". Am I missing something?

Reply:
While "allowed origins" is plural, it's referring to the allowed origins property and there's one of those.

Reply:
@wilkinsona Thanks for the explanation! I reverted it.